### ISSUE TITLE
Samples.pkl not found

### ISSUE THREAD

**Issue by aditya-AI** [arnaldog12/Deep-Learning#2]
Hi Arnald, the data path you have provided returns a samples.rar which has sample_train.csv and sample_test.csv.

Could you please tell me how do I get the actual samples.pkl you used in your Keras.ipynb notebook?

**Comment by chinar-amrutkar** [comment]
Yup, facing the same issue. Have you found any alternatives to this repo for this same functionality(head pose/position) for the time being?

Thanks.

**Comment by arnaldog12** [comment]
Sorry for that. I'll generate the samples.pkl file again for you, guys. I'll try to make it today.

**Comment by arnaldog12** [comment]
You can download **samples.pkl** [here](https://we.tl/t-4aQmNrpO9T).
The dump has already the dataset divided in train/test. So, you can call like this:
```py
x_train, x_test, y_train, y_test = pkl.load(open('data/sample.pkl', 'rb'))
```

**Comment by chinar-amrutkar** [comment]
Thanks Arnaldo!

**Comment by sf308** [comment]
I am unable to access the following link:
https://we.tl/t-4aQmNrpO9T

I tried importing dat in csv format from this source https://drive.google.com/file/d/1QPgrmOtJ2Y1jD0-0Ya6P4GUsyGDJKVtZ/view?usp=sharing, merging the two csv files together then training and testing but I end up getting not very good results:
 Roll: -959.48°
Pitch: -1338.22°
  Yaw: -698.99°

**Comment by arnaldog12** [comment]
I uploaded a new samples.pkl inside a .rar file available [here](https://drive.google.com/open?id=1lxwO-A-VBZCVuKKP1FgE_h0ZgfxUvlR4).

**Comment by arnaldog12** [comment]
@sf308,
are you normalizing (StandardScaler, MinMaxScaler) the training and test sets on the same way?

**Comment by UmarZubair** [comment]
Hi there, I did exactly as you did but still getting bad results on the same lena.png file.
Roll: -2175.33°
Pitch: 3279.93°
Yaw: -45.37°
I can't understand what the fault here is. Can you help me in resolving this.

**Comment by arnaldog12** [comment]
Can you run the notebook from beginning up to the end, @UmarZubair?

**Comment by msvargas** [comment]
@arnaldog12 Hi, no working :(
Results:
 Roll: 2262.16°
Pitch: 1977.92°
  Yaw: 1040.01°

**Comment by drevillanueva** [comment]
Also got bad results after following exactly what you did using the data that you've uploaded. Can you also upload the same data that you used for training and testing?

**Comment by sandhyacs** [comment]
@arnaldog12 , 

Thank you for this nice project, it helped me a lot. But I want to know how you are creating sample_train.csv and sample_test.csv ?

**Comment by arnaldog12** [comment]
Please, take a look at my [Medium article](https://medium.com/analytics-vidhya/face-pose-estimation-with-deep-learning-eebd0e62dbaf), @sandhyacs
